test/edu/csus/ecs/pc2/ui/SubmissionSolutionListTest.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui;
+
+import java.io.File;
+import java.util.List;
+
+import edu.csus.ecs.pc2.core.util.AbstractTestCase;
+
+/**
+ * Unit test.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class SubmissionSolutionListTest extends AbstractTestCase {
+
+    public void testgetAllDirectoryEntries() {
+
+        String cdpDir = "samps/contests/tenprobs";
+        cdpDir = "c:\\test\\current\\samps\\contests\\tenprobs";
```

Comment:
Unit test files should not have hard-coded references to Windows-path files. 
(I realize that the following `if` statement will keep this unit test from failing on a GitHub/Gitlab/Unix build, but still...